XamlAnimatedGif.Shared/Animator.cs

```diff
@@ -346,7 +346,7 @@ private static byte[] CreateIndexStreamBuffer(GifDataStream metadata, Stream str
                         CopyToBitmap(lineBuffer, _bitmap, offset, bufferLength);
                     }
 #if WPF
-                    var rect = new Int32Rect(desc.Left, desc.Top, desc.Width, desc.Height);
+                    var rect = new Int32Rect(desc.Left, desc.Top, desc.Width - desc.Left, desc.Height - desc.Top);
```

Comment:
I don't think this is correct. `Width - Left` doesn't represent anything meaningful. The `Int32Rect` expects a width, so we should pass a width.

I think the problem is that this image is corrupted. The frame top + frame height is greater than the total height of the image (specified in the logical screen descriptor). The fix should be to use `Math.Min` to limit the width and height of the dirty rect.